lib/NormalModule.js

```diff
@@ -178,13 +178,13 @@ class NormalModule extends Module {
 		return loaderContext;
 	}
 
-	getCurrentLoaderName(loaderContext) {
+	purifyCurrentLoaderName(loaderContext) {
```

Comment:
`purifyCurrentLoaderName` => `getLoader || getLoaderName || getCurrentLoader` 


```js
getLoader(loaderContext) {
  if (this.loaders && this.loaders.length) {
    // Why is origin needed here (fallback?) ?
    const { loader } = this.loaders[this.loaderIndex]

    return loader
  }
  
  return null
}
```

Reply:
```javascript
 const { loader } = this.loaders[loaderContext.loaderIndex]
```

Here the `loader` is an absolute path and seems too long.

If there is a config like:
```javascript
rules:[
  {
    test:/\.jsx$/,
    loader: "babel-loader",
  }
]
```

Then I prefer to show `babel-loader` than `/User/xxx/projects/node_modules/babel-loader/index.js` in the message. So that why I add an `origin` prop and named the `purifyCurrentLoaderName ` API.

Reply:
`loader` returns an absolute path ? kk... then the logic to get only the name needs to stay of course :). I thought this is only needed in case `origin` is use the determined the loader name (I didn't test it myself yet)

Reply:
```
return loader.split('/').find((p) => p.includes('-loader'))
```

The current code won't work with most loaders (`filename !== 'loaderName'`) e.g `postcss-loader` would display 'index.js' as the 'loaderName' atm